wiki/Game_modifier/Hard_Rock/en.md

```diff
@@ -57,7 +57,11 @@ Due to this much larger spike in difficulty compared to other [game modes](/wiki
 
 *Notice: Using this mod on osu!mania will result in an unranked play.*
 
-In [osu!mania](/wiki/Game_mode/osu!mania), only the judgement difficulty (OD) and the AR values are increased. All other difficulty settings are left as-is. Note placement is not affected.
+In [osu!mania](/wiki/Game_mode/osu!mania), the HP value are increased by 40% while the timing windows are 40% stricter. Though the AR value is displayed as increased, it does not affect the gameplay.
```

Comment:
if the conversion is `hit window / 1.4`, then you'd say it is about 29% stricter (`1 - 1 / 1.4 ≈ 0.2857`)

```suggestion
In [osu!mania](/wiki/Game_mode/osu!mania), the HP value is increased by 40% while the timing windows are about 29% stricter. Though the AR value is displayed as increased, it does not affect the gameplay.
```